python/servo/build_commands.py

```diff
@@ -764,6 +772,34 @@ def gstreamer_root(target, env):
         return None
 
 
+def angle_root(target):
+    arch = {
+
+        "aarch64": "arm64",
+        "x86_64": "x64",
+    }
+
+    angle_x64 = arch[target.split('-')[0]]
+    print("angle_x64 Architecture = {}".format(angle_x64))
+    angle_default_path = path.join(os.getcwd(),
+                                   "support\\hololens\\packages\\ANGLE.WindowsStore.Servo.2.1.13\\bin\\UAP\\")
+    angle_default_path += angle_x64
```

Comment:
We already have code to create this path in https://github.com/servo/servo/blob/91469aa47215c8e1b4e4ae5643ca793b3f65af9b/python/servo/build_commands.py#L295-L298, so let's use that code in this method (since the use of path.join there is a bit easier to read) and call angle_root in that code to avoid duplication.

Reply:
Done.